tests/osVersionRCompare.coffee

```diff
@@ -0,0 +1,40 @@
+# we keep this to ensure balena-semver has the exact same behavior
+# and will drop it in a later PR
+semver = require('semver')
+
+safeSemver = (version) ->
+	version.replace(/(\.[0-9]+)\.rev/, '$1+rev')
+
+getRev = (osVersion) ->
+	rev = semver.parse(osVersion).build
+	.map((metadataPart) -> /rev(\d+)/.exec(metadataPart)?[1])
+	.filter((x) -> x?)[0]
+
+	if rev?
+		parseInt(rev, 10)
+	else
+		0
+
+isDevelopmentVersion = (version) ->
+	/(\.|\+|-)dev/.test(version)
+
+# TODO: Drop this once we make sure balena-semver is doign the exact same thing
+exports.osVersionRCompare = (versionA, versionB) ->
```

Comment:
I assume if all the tests pass using `balena-semver`, we can go ahead and remove this now, as it doesn't serve any further purpose. This only needs to be tested once (which is already done by this PR), so I guess that is what you meant to drop it once it does the same thing.

Reply:
I thought that I should have a merged PR as a proof first :)
But yea I can add an extra commit for that and rely have the appveyor build as a prood if you prefer it this way :)
Just give a note!

Reply:
Not really important, anyway is fine, whatever it is easier for you. Maybe having a follow-up PR is cleaner and provides a bit more searchability (doubt it will matter, but still)